### PR TITLE
Fix UG bugs

### LINKED ISSUE #146: [PE-D] Incorrectly hyperlinked link in Quick Start section
The hyperlink in "Download the latest propertywhiz.jar from *here*" directs me to the user guide website and not the team's product.

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: skythefire/ped#1

### LINKED ISSUE #147: [PE-D] UG \| syntax
\| syntax is not explained in the UG. Seems like it means 'or'. Maybe could add a how to use user guide at the start

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: hernpiblo/ped#1

### LINKED ISSUE #150: [PE-D] Grammatical error in Quick Start
Step 5 of User Guide: "Enter `clear` start from scratch", should this be "Enter `clear` *to* start from scratch"?

There are several "\"s in the Command Summary section. I am unsure if they are unintentional.

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: skythefire/ped#2

### LINKED ISSUE #154: [PE-D] Header different from link
link says valid input but header says valid fields. perhaps can standardize

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: hernpiblo/ped#3

### PR DESCRIPTION
**Pull request by trxe**
closes #154
closes #150
closes #147
closes #146

PREVIEW: https://trxe.github.io/tp/UserGuide.html

- Corrects all `\|` in command summary
- Fix wrong/broken links
- Some better titles
- References to histogram changed to bar chart (#157)
- `properties` and `buyers` changed to `property` and `buyer` document-wide with a disclaimer that they are interchangeable.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -401,6 +419,10 @@ Example csv output of `export buyer`
 "Frail Duckie","93234267","fail@mega.com","420003","tiny,1rm"
 ```
 
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
+Try copying these to a spreadsheet and saving it as a .csv file. Then import them as sample inputs!
```
Comment (waynetee) [inline docs/UserGuide.md:423]:
```suggestion
Try copying these to a .csv file. Then import them as sample inputs!
```

**Review by waynetee — APPROVED** [review]
Thanks for the changes!

**Comment by waynetee** [comment]
Btw if you want to close issues automatically with your pr description you need to list them out one by one.

```
closes ABC
closes DEF
```

File: docs/UserGuide.md [hunk 2]
```diff
@@ -42,27 +42,27 @@ PropertyWhiz (PropertyWhiz) is a **desktop app for managing properties and prope
...
 **Exit** | `exit`
 **Help** | `help`
-**Sort** | `sort (properties \| buyers) (price \| name) (asc \| desc)`
-**Match** | `match ( auto \| property INDEX \| buyer INDEX )`
-**Import** | `import (properties \| buyers)`
-**Export** | `export (properties \| buyers)`
+**Sort** | `sort ( property | buyer ) (price | name) (asc | desc)`
+**Match** | `match ( auto | property INDEX | buyer INDEX )`
+**Import** | `import (property | buyer)`
+**Export** | `export (property | buyer)`
```
[the remark this answers is not in the document]
Comment (trxe) [reply docs/UserGuide.md:65]: We could standardise the in app help messages to use 'property' and 'buyer' too

[the remark this answers is not in the document]
Comment (waynetee) [reply docs/UserGuide.md:65]: I thought it was a feature that for some commands, the plural form was more intuitive, like find buyers and export properties

File: docs/UserGuide.md [hunk 3]
```diff
@@ -127,18 +131,30 @@ TODO: Define named parameters, positional parameters
...
+### Valid properties/buyers
 Here are some fields that are shared amongst commands.
 
 #### Property/Buyer names
 * Names must start with a letter or number, and only contain alphanumerical characters, spaces and hyphens (`-`).
 * Names have a maximum allowed length of 50.
+* Buyers with identical names are not allowed.
+
+#### Property addresses
+* Addresses must start with a letter or number, and only contain alphanumerical characters, and the following special characters:
```
[the remark this answers is not in the document]
Comment (trxe) [reply docs/UserGuide.md:150]: Yep

File: docs/UserGuide.md [hunk 4]
```diff
@@ -127,18 +131,30 @@ TODO: Define named parameters, positional parameters
 
 * Extraneous parameters for commands that do not take in parameters (such as `help`, `list`, `exit` and `clear`) will be ignored.<br>
   e.g. if the command specifies `help 123`, it will be interpreted as `help`.
+  
+* As many of our commands use slashes `/`, if possible do not use `/` in your inputs as it may lead to unexpected results.
```
[the remark this answers is not in the document]
Comment (trxe) [reply docs/UserGuide.md:135]: I wasn't sure about how to phrase this, but there is no argument in which the argument itself can contain /

**Pull request merged**